unison-cli/unison/ArgParse.hs

```diff
@@ -244,15 +245,17 @@ codebasePathParser :: Parser (Maybe CodebasePathOption)
 codebasePathParser = do
     optString <- optional . strOption $
          long "codebase"
-      <> metavar "codebase/path"
+      <> short 'c'
+      <> metavar "CODEBASE/PATH"
```

Comment:
what's with the all caps metavar vs lower case?

Reply:
The rest of the metavars in this file are uppercase;
It's a relatively common convention in modern cli's, open for debate of course, but I find it helps make it clear when something is supposed to be replaced.

I've also commonly seen `<METAVAR>` with angle brackets.

I'm open to whatever we think is best, but this is how stack does it 😄 

After a quick sampling,

* git uses `<metavar>`
* rg uses `<METAVAR>`
* stack uses `METAVAR`
* fzf uses `METAVAR`

So looks like conventions are all over the place these days 🤷🏼‍♂️ ;
The most important is that we're consistent I think. If I had to pick I'd prefer `<METAVAR>`, but right now the other unison flags use `METAVAR`

Reply:
Interesting...